CSS/Components/navbar.css

```diff
@@ -0,0 +1,25 @@
+.nav-container {
+  width: 100%;
+}
+.navbar-container {
+  background-color: var(--primary-color);
+  color: var(--off-white);
+  display: flex;
+  justify-content: space-between;
```

Comment:
`justify-content: space-between` may cause problems with logo responsive as told by Vikram sir.
Try to do it with `margin-left: auto`

Reply:
Thanks Tanishq,Made the changes